src/unity/python/turicreate/test/test_boosted_trees.py

```diff
@@ -127,7 +127,7 @@ def test__list_fields(self):
         """
         model = self.model
         fields =  model._list_fields()
-        self.assertEqual(set(fields), set(self.fields_ans))
+        self.assertTrue(set(self.fields_ans).issubset(fields))
```

Comment:
Is the addition of new fields intentional then? (The context provided for this PR only mentioned avoiding re-computation of predictions.)

If it is intentional, shouldn't we document the new expected fields, both in the test and in the Pydoc?

Reply:
Yes, I agree.  I can add that documentation.